app/models/services/route_binding.rb

```diff
@@ -1,3 +1,5 @@
+require 'cloud_controller/process_route_handler'
```

Comment:
Does this relate to the change in spec helper?

Reply:
yes!